gym/lib/gym/generators/build_command_generator.rb

```diff
@@ -129,7 +129,8 @@ def archive_path
 
       def result_bundle_path
         unless Gym.cache[:result_bundle_path]
-          path = File.join(Gym.config[:output_directory], Gym.config[:output_name]) + ".result"
+          path = Gym.config[:result_bundle_path]
+          path ||= File.join(Gym.config[:output_directory], Gym.config[:output_name] + ".result")
```

Comment:
Does putting the `.result` inside here make any real difference or is this just nicer or more consistent with how `File.join` should be used?

Reply:
I'm not sure what exactly are you referring to, the result directory name should be `#{output_dir}/#{output_name}.result`. `File.join` adds the file separator between all the parameters. I couldn't do `File.join(our_dir, out_name, ".result)` as that would create `#{output_dir}/#{output_name}/.result`